Firestore/core/test/firebase/firestore/testutil/time_testing.h

```diff
@@ -27,6 +27,10 @@ namespace testutil {
 
 using time_point = std::chrono::system_clock::time_point;
 
+/** Returns the current time, in milliseconds. */
+std::chrono::time_point<std::chrono::steady_clock, std::chrono::milliseconds>
```

Comment:
I think this was previously defined in `async_tests_util.h` (called `now`), should the previous version of this function now be deleted?

Reply:
A subsequent PR removes async_tests_util.h altogether in the interest of making these utilities more broadly available that just the AsyncQueue and Executor tests.